integrations/sklearn/Neptune-Scikit-learn.ipynb

```diff
@@ -0,0 +1,2401 @@
+    "id": "AW4MM4qj5ErX",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Step 3: Create an Experiment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "This creates an experiment in Neptune.\n",
+    "\n",
+    "Once you have a live experiment you can log things to it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "neptune.create_experiment(name='regression-example',\n",
+    "                          tags=['RandomForestRegressor', 'regression'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "Click on the link above to open this experiment in Neptune.\n",
+    "\n",
+    "For now it is empty but keep the tab with experiment open to see what happens next."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "AW4MM4qj5ErX",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Step 4: Log regressor summary"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 71
+    },
+    "colab_type": "code",
+    "id": "GvDSBSrOx-R4",
+    "outputId": "667d1b2f-cdef-44f0-f505-832205dea223",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "from neptunecontrib.monitoring.sklearn import log_regressor_summary\n",
+    "\n",
+    "log_regressor_summary(rfr, X_train, X_test, y_train, y_test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "You just logged information about the regressor, including:\n",
+    "\n",
+    "* [logged regressor parameters](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/details) as properties,\n",
+    "* [logged pickled model](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/artifacts?path=model%2F&file=estimator.skl),\n",
+    "* [logged test predictions](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/artifacts?path=csv%2F&file=test_predictions.csv),\n",
+    "* [logged test scores](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/charts),\n",
+    "* [logged regressor visualizations](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/logs) - look for \"charts_sklearn\",\n",
+    "* [logged metadata](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/details) including git summary info."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "tests"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# tests\n",
+    "exp = neptune.get_experiment()\n",
```

Comment:
I think it is safer to divide tests into two cells and have `neptune.stop()` right after this.
It will ensure that all of those objects are logged.

Reply:
done